docs/README.md

```diff
@@ -32,7 +36,7 @@ Before generating your store's sitemap, you might want to adjust if products, na
 
 6. From the dropdown list, choose the `vtex.routes-bootstrap@0.x` app.
 
-7. If this is **not** the **first time** you're generating your store's sitemap or if your store's routes **did not suffer any changes** since the last time you generated your store's sitemap, go to step 8. Otherwise, run the following query:
+7. If this is **not** the **first time** you are generating your store's sitemap or if your store's routes **did not suffer any changes** since the last time you generated your store's sitemap, go to step 8. Otherwise, run the following query:
```

Comment:
```suggestion
7. If this is **not the first time** you are generating your store's sitemap or if your store's routes **did not suffer any changes** since the last time you generated your store's sitemap, go to step 8. Otherwise, run the following query:
```